packages/ilios-common/package.json

```diff
@@ -81,6 +82,7 @@
     "@ember-data/tracking": "5.3.8",
     "@popperjs/core": ">= 2.1.0",
     "class-validator": ">= 0.14.0",
+    "ember-local-storage": "^2.0.7",
```

Comment:
Because it is a v1 addon, this probably isn't needed as production dependency adding get pushed into the consuming app by embroider.

Reply:
ah ok. removed. :heavy_check_mark: